common/patches/titanium.cpp

```diff
@@ -1421,7 +1535,7 @@ namespace Titanium
 			eq->Class[char_index] = 0;
 			eq->Face[char_index] = 0;
 
-			strncpy(eq->Name[char_index], "<none>", 6);
+			strn0cpy(eq->Name[char_index], "<none>", 6);
```

Comment:
Sorry I missed this one, looks like an accidental change. This should be reverted too (it would replace the last '>' with a null)

Reply:
Aye, missed this one.  Updated.